packages/app-store/zoomvideo/api/callback.ts

```diff
@@ -83,5 +84,5 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
     },
   });
 
-  res.redirect("/apps/installed");
+  res.redirect(getInstalledAppPath({ variant: "conferencing", slug: "zoom" }));
```

Comment:
This is to support redirecting to the correct category within the Installed Apps page once the app has been installed.